sktime/classification/kernel_based/_rocket_classifier.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 """RandOm Convolutional KErnel Transform (ROCKET)."""
 
-__author__ = "Matthew Middlehurst"
+__author__ = ["Matthew Middlehurst", "Viktor Dremov"]
```

Comment:
the strings should be GitHub IDs.
I know it's inconsistent currently, but later we'll want to be using GitHub IDs.

Reply:
96c657bb